build.gradle.kts

```diff
@@ -188,6 +188,14 @@ allprojects {
             if (file("../build.local.properties").exists()) {
                 testRuntimeOnly(files("../build.local.properties"))
             }
+            // CI platform can populate these files to impact the respective task cache key.
+            // For example, adding the PG server information to ensure tests are not considered cached across different versions.
```

Comment:
Please use regular input.property('pg.version', ...) instead.

Reply:
Where does that get populated and what value would that have? Is that supposed to be the `PG_VERSION` matrix value? If so then it does not work because the value would only be the major versions, e.g. "10". It would not reflect patch version changes as new versions are released and the major version Docker tag gets repointed to newer releases.